pipelines/ocw/docker-image-pipeline.yml

```diff
@@ -0,0 +1,33 @@
+---
+resources:
+    - name: ol-infrastructure
+      type: git
+      source:
+          uri: https://github.com/mitodl/ol-infrastructure.git
+          branch: ((ol-infrastructure-git-ref))
+    - name: node-image
+      type: docker-image
+      check_every: 1h
+      source:
+          repository: node
+          tag: '14'
```

Comment:
To cut down on the amount of data being pulled it might make sense to base our image build off of `14-buster-slim`

Reply:
Wow, it's a lot smaller:

```
(venv) ol-infrastructure$ docker image ls
REPOSITORY                    TAG                 IMAGE ID            CREATED             SIZE
[ ... ]
node                          14                  9153ee3e2ced        8 days ago          943MB
node                          14-buster-slim      d4c353c8e216        8 days ago          181MB
```

Reply:
dd2b716b changes the base image.